tools/clang/lib/SPIRV/DebugTypeVisitor.cpp

```diff
@@ -291,7 +290,9 @@ DebugTypeVisitor::lowerToDebugType(const SpirvType *spirvType) {
     SpirvDebugInstruction *elemDebugType =
         lowerToDebugType(matType->getElementType());
     debugType = spvContext.getDebugTypeArray(
-        spirvType, elemDebugType, {matType->numRows(), matType->numCols()});
+        spirvType, elemDebugType,
+        llvm::SmallVector<uint32_t, 2>(
```

Comment:
I don't think you need to have `llvm::SmallVector<uint32_t, 2>(` here. Initialization by `{...}` should work.
https://en.cppreference.com/w/cpp/language/list_initialization

Reply:
done